### ISSUE TITLE
 ERROR: for web Cannot link to a non running container

### ISSUE THREAD

**Issue by diogocsc** [libresh/wordpress#6]
After editing the env file with:
![docker_compose_up_wordpress_indie_env](https://cloud.githubusercontent.com/assets/1706650/18755915/c072bcd0-80e5-11e6-8ed3-86d6c1d4870f.png)

ran docker-compose up, andd got ERROR.

![docker_compose_up_wordpress_indie](https://cloud.githubusercontent.com/assets/1706650/18755903/b75a17ba-80e5-11e6-8e23-0afb2981f4ef.png)

and docker-compose up db got:

![docker_compose_up_db_wordpress_indie_env](https://cloud.githubusercontent.com/assets/1706650/18756022/3796b1b8-80e6-11e6-9bc7-f2d3a1ded21e.png)

**Comment by diogocsc** [comment]
Meanwhile, noted that env file is not well written.
If on env file i replace : by = then it goes further...

![Uploading docker_compose_up_logs_wordpress_indie_env.png…]()

**Comment by pierreozoux** [comment]
What is docker version, compose version, OS version?

What is Internet saying about this error?

**Comment by diogocsc** [comment]
![versionsdocker](https://cloud.githubusercontent.com/assets/1706650/18790894/87fa34c8-81a8-11e6-8278-c9941e4235ad.png)

OS where I am running this is Windows 7 home

**Comment by pierreozoux** [comment]
Can you again, to be sure, `docker-compose up db` and show the logs.

If it works, then from anothe terminal, do: `docker-compose up app`.

This should work I don't see why it doesn't work.

**Comment by diogocsc** [comment]
db ups fines. app with problem connecting to db...

![docker_wp_indie](https://cloud.githubusercontent.com/assets/1706650/18798352/05dfc9c2-81ca-11e6-8a9a-dfaa0280ba10.png)

**Comment by diogocsc** [comment]
After several combinations I managed to go through without apparent errors.
I needed to update compose, replacing links: - db:mysql with just links: - db
I also needed to have all the env variables.
This said, in the end, when i try to access through browser, cannot connect
I'm running kitematic at same time to check the port.

![apparentrunning](https://cloud.githubusercontent.com/assets/1706650/18801393/6437f4ee-81da-11e6-8765-aa46b83e841a.png)

![env](https://cloud.githubusercontent.com/assets/1706650/18801400/6a82faa6-81da-11e6-84ae-d28ba61ddd30.png)

![kitematic_port](https://cloud.githubusercontent.com/assets/1706650/18801430/8d192360-81da-11e6-8f08-2bd8dd5df5b8.png)

![firefox](https://cloud.githubusercontent.com/assets/1706650/18801511/f36e847a-81da-11e6-9842-9ca23ffc25f3.png)

I've also tried to add port directly to initial file, without success...
